server/routes/users.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
```

Comment:
@KovalivNataliia  you can import express and other libraries via import statement: import * as express from 'express';